### PR TITLE
Added Findbugs JSR305 annotations library

### PR DESCRIPTION
**Pull request by triller-telekom**
Signed-off-by: Stefan Triller <stefan.triller@telekom.de>

### CODE REVIEW

File: third-party/pom.xml [hunk 1]
```diff
@@ -143,6 +143,12 @@
                   <transitive>false</transitive>
                   <source>false</source>
                 </artifact>
+                <!-- https://mvnrepository.com/artifact/com.google.code.findbugs/jsr305 -->
+                <artifact>
+                    <id>com.google.code.findbugs:jsr305:jar:3.0.2</id>
+                    <transitive>false</transitive>
+                    <source>false</source>
```
Comment (htreu) [inline third-party/pom.xml]: Are there sources available? It would be nice to have them in the IDE for reference (even on annotations).

**Comment by sjka** [comment]
Hrmmm... Contributes to #3624
CQ 13833 (PB CQ 11244) already created (but not yet approved).

**Comment by sjka** [comment]
Good news: The CQ was finally approved.

**Comment by sjka** [comment]
Thanks!

**Pull request merged**